test_scripts/API/VehicleData/StabilityControlsStatus/001_GetVehicleData_StabilityControlsStatus.lua

```diff
@@ -10,7 +10,7 @@
 -- 4) App is activated
 
 -- Steps:
--- 1) App send GetVehicleData (with stabilityControlsStatus = true) request to SDL
+-- 1) App sends GetVehicleData (with stabilityControlsStatus = true) request to SDL
 -- SDL does:
 --  - send VehicleInfo.GetVehicleData (with stabilityControlsStatus = true) request to HMI
```

Comment:
@ydementieiev sendS here and in all cases below

Reply:
@LitvinenkoIra done